README.md

```diff
@@ -56,10 +61,41 @@ op.attempt(next => {
 
 ```
 
+### `Async/await` Support
+
+Retryme now supports `async/await`. It accepts three arguments:
+
+- `retry`: The configuration for create a new [retryme instance][retryme-instance].
+- `attempt`: The real `async` function which will be retried with `await`.
+- `ignore`: The `ignore` function also for setup [retryme instance][retryme-instance] to terminate retries.
+
+A simple example of how to use it is as follows:
+
+```js
+const awaitRetryme = require('retryme/await');
+
+function some_async_func(val) {
```

Comment:
since we cannot pass in values for this function, we have to assume that the outer function takes no arguments and is wrapping a thenable or promise or async function.